### ISSUE TITLE
Export ToastElement?

### ISSUE THREAD

**Issue by supermonkeybrainz** [jossmac/react-toast-notifications#104]
It looks like `ToastElement` provides the majority of the functionality I want, particularly the transition and mouse events.  Could we just export that?  I don't want anything the `DefaultToastElement` exports, just the `ToastElement`.

**Comment by supermonkeybrainz** [comment]
If I am granted permission, I can create a pull request for this.  Literally just adding an `export` before the `const ToastElement` in `ToastElement.jsx`.  The library is great, just want to be able to show _only the element I pass_.  Then things would be perfect!  Let me know if this is possible.

**Comment by supermonkeybrainz** [comment]
Would be even better if we could get rid of the text/background class styles on that container.  Can work around that, but would be awesome if we could drop those too (move to a wrapper around `DefaultToast`?).

**Comment by jossmac** [comment]
@supermonkeybrainz could you describe the problem you're trying to solve?

**Comment by supermonkeybrainz** [comment]
I want to be able to display a generic toast element.  One with no preconceptions of how it should look.  I want the animations/placement/timeout/close behaviors, but I don't necessarily want the library to define anything beyond an empty container in which to render my own component.

Specifically, I need to integrate with Carbon Design System using their notification styling:  https://react.carbondesignsystem.com/?path=/story/notifications--toast

I have done this successfully with this library, but I had to copy a bunch of things down to my codebase to get just the container with the behavior.  Would be nice to just export the `ToastElement`, which is almost exactly what I want (not quite, but close enough for my purposes).  Let me know if that doesn't make sense.

Thanks!

**Comment by rohanx96** [comment]
Yeah, I need to customise the look of the toast as well. I feel it limits the usage of an otherwise great library. Exporting `ToastElement` should be a quick fix to allow customising the entire look of the notification rather than just the message content.

**Comment by arosiclair** [comment]
I'd also very much like to re-use the transition functionality from ToastElement without having to copy the source.
